libraries/botframework-config/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "botframework-config",
+  "author": "Microsoft Corp.",
+  "description": "library for working with Bot Framework .bot configuration files",
+  "version": "4.0.0-preview1.2",
+  "license": "MIT",
+  "keywords": [
+    "bots",
+    "chatbots"
+  ],
+  "main": "./lib/index.js",
+  "typings": "./lib/index.d.ts",
+  "files": [
+    "lib"
+  ],
+  "engines": {
+    "node": ">=8.0"
+  },
+  "homepage": "http://github.com/Microsoft/botbuilder-js",
+  "bugs": {
+    "url": "http://github.com/Microsoft/botbuilder-js/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:Microsoft/botbuilder-js.git"
+  },
+  "devDependencies": {
+    "@types/uuid": "^3.4.3",
+    "mocha": "^5.0.5"
+  },
+  "dependencies": {
+    "linq-collections": "^1.0.254",
```

Comment:
This dependency is heavy in comparison to other more lightweight options. [node-klaw](https://github.com/jprichardson/node-klaw) is trusted, extremely lightweight and more widely know in the JS community.  We might consider this when taking on dependencies.